packages/core-js/utils/hardhat.js

```diff
@@ -0,0 +1,30 @@
+async function getContractAST({ sourceName, contractName }) {
```

Comment:
Also created this util for hardhat related operations.

For now it only contains utils to retrieve AST data.